### PR TITLE
Change class name of button (fix #993)

### LINKED ISSUE #993: Organ mutations aren't updated
I don't think it is related to #925 but I noticed 'new' organ mutations aren't shown on the committee pages. 
See e.g. [Kas Kontrole Kommissie 2020](https://www.gewis.nl/organ/show/273).
[The mutations from AV172](https://www.gewis.nl/decision/AV/172) are not shown.

### PR DESCRIPTION
**Pull request by armandduijn**
See #933

### CODE REVIEW

**Comment by jszanto** [comment]
Did you test this? Afaik changing the name of that class only there will break the button.

**Comment by armandduijn** [comment]
Nope. I only find and replaced occurrences of the class name. I’ll look into it.

**Comment by armandduijn** [comment]
The functionality has been fixed and tested. The button is now displayed and toggles the metadata modal. See commit body and changes for full details.

File: public/js/photoswipe-ui-default.min.js [hunk 1]
```diff
@@ -501,7 +501,7 @@
                     name: 'button--info',
                     option: 'infoEl',
                     onTap: function() {
-                        console.log(_captionContainer, 'caption--empty', pswp.currItem.title);
+                        _toggleShareModal();
```
Comment (kokx) [inline public/js/photoswipe-ui-default.min.js]: Usually, the leading `_` means that the function used is private, or somehow not part of the public API.  Or that it hides functionality that should not be relied upon as a user of the API. Is that the case here?

Comment (armandduijn) [reply public/js/photoswipe-ui-default.min.js]: Yes, it's a private function but it's being called from within the library. I edited the library's code. Unfortuantly, there isn't a way to toggle the modal from the public API.

Right now the metadata modal piggy-backs on the share modal provided by the library. If modifying the library's code to toggle the share modal is undesirable, we would have to implement a custom modal.

Comment (armandduijn) [reply public/js/photoswipe-ui-default.min.js]: @kokx Do you have any comments?

Comment (kokx) [reply public/js/photoswipe-ui-default.min.js]: Yeah. That I hate that something like this is necessary. But if it absolutely is necessary, its okay I guess.

**Comment by armandduijn** [comment]
@Koen1999 No more blocking comments. Ready to merge.

**Pull request merged**